xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -5265,9 +5266,12 @@ glusterd_import_friend_volumes_synctask(void *opaque)
     conf->restart_bricks = _gf_true;
 
     while (i <= count) {
```

Comment:
I've just seen that `i` is initialized to 1. Is this consistently done everywhere ? otherwise the bitmap could be incorrectly interpreted.

Reply:
Yes we always start from 1 first volume name is volume1.<keys-suffix>